lib/tasks/install_secrets.rake

```diff
@@ -30,13 +30,17 @@ task :install_secrets, [] do
 
   secrets = {}
 
+  # When we get parameters from the store, we want to ignore the env_name
+  # and the parameter namespace.  Calculate how many levels there are here
+  num_ignored_key_levels = [env_name, namespace].join('/').split('/').count
```

Comment:
should these 2 "utils" be put into a utility method so it can be unit tested (and potentially reused?)